cranelift/zkasm_data/benchmarks/sha256/generated/helpers/2-exp.zkasm

```diff
@@ -0,0 +1,65 @@
+two_power:
```

Comment:
Should probably use a symbol name that's not as likely to collide with something user-defined if/once we start outputting that to the outputs. `__foo` has been in common use for this purpose elsewhere as `__` is reserved for internal use by the likes of the C standard and/or the standard libraries.

Reply:
Good idea, filed https://github.com/near/wasmtime/issues/157